tests/test_georaster.py

```diff
@@ -209,46 +209,83 @@ def test_inters_img(self):
 
     def test_interp(self):
 
+        # FIRST, we try on a Raster with a Point interpretation in its "AREA_OR_POINT" metadata: values interpolated
+        # at the center of pixel
         r = gr.Raster(datasets.get_path("landsat_B4"))
+        assert r.ds.tags()['AREA_OR_POINT'] == 'Point'
 
         xmin, ymin, xmax, ymax = r.ds.bounds
 
-        # testing interp, find_value, and read when it falls right on the coordinates
+        # We generate random points within the boundaries of the image
         xrand = (np.random.randint(low=0, high=r.ds.width, size=(10,))
                  * list(r.ds.transform)[0] + xmin + list(r.ds.transform)[0]/2)
```

Comment:
Why not have `r.ds.transform[0]*1.5` instead of `list(r.ds.transform)[0] + list(r.ds.transform)[0]/2`?

Reply:
Removed